packages/webapp/src/containers/Task/TaskReadOnly/useReadonlyTask.js

```diff
@@ -29,10 +29,10 @@ const getManagementPlanTile = (managementPlan, tasksByManagementPlanId) =>
 
 export const useReadonlyTask = (task_id) => {
   const task = useSelector(taskWithProductSelector(task_id));
-  if (task === undefined) return undefined;
   const tasksByManagementPlanId = useSelector(taskEntitiesByManagementPlanIdSelector);
 
   const getTransplantTask = (task) => {
+    if (task === undefined) return undefined;
```

Comment:
I don't think it's possible to reach here with task `undefined` because of your optional chaining and the ternary expression -- right?

I think maybe I _do_ prefer putting `if (task === undefined) return undefined;` right into the first line of the useMemo 😆 It is a bit clearer!